src/main/kotlin/io/github/gunpowder/commands/VanishCommand.kt

```diff
@@ -51,12 +51,12 @@ object VanishCommand {
 
         // Sending info to player
         ctx.source.player.sendMessage(
-                LiteralText(
+                TranslatedText(
```

Comment:
Optionally look into using the Text builder here, iirc it supports translated text

Reply:
I don't think Text builder is appropriate here. These two messages are mostly independent.
I though of using concatenate to construct the translate key, but I think the current structure makes sense and is good enough.